source/operations/install-deploy-manage/deploy-minio-single-node-multi-drive.rst

```diff
@@ -40,12 +45,10 @@ Local JBOD Storage with Sequential Mounts
    For deployments that *require* using network-attached storage, use
    NFSv4 for best results.
 
-Deploy Standalone Multi-Drive MinIO
------------------------------------
+Deploy Single-Node Multi-Drive MinIO
+------------------------------------
 
-The following procedure deploys MinIO in :guilabel:`Standalone Mode` consisting
-of a single MinIO server and a single drive or storage volume. Standalone
-deployments are best suited for evaluation and initial development environments.
+The following procedure deploys MinIO consisting of a single MinIO server and a single drive or storage volume.
 
 .. admonition:: Network File System Volumes Break Consistency Guarantees
```

Comment:
This admonition is really repetitive to the one just above it on the page. (Lines 36-44 are basically the same, and there's just a heading and one line paragraph between them.)